src/code42cli/cmds/profile.py

```diff
@@ -124,12 +133,13 @@ def update(name, server, username, password, disable_ssl_errors):
 
 @profile.command()
 @profile_name_arg()
-def reset_pw(profile_name):
+@debug_option
+def reset_pw(profile_name, debug):
     """\b
     Change the stored password for a profile. Only affects what's stored in the local profile,
     does not make any changes to the Code42 user account."""
     password = getpass()
-    profile_name_saved = _set_pw(profile_name, password)
+    profile_name_saved = _set_pw(profile_name, password, debug)
```

Comment:
It seems there are missing tests for the cases when passing `debug` and `totp` to these profile commands that show that those arguments are used when creating the sdk

Reply:
good catch